packages/asset-swapper/src/utils/quote_report_generator.ts

```diff
@@ -17,13 +18,15 @@ export interface BridgeReportSource {
     liquiditySource: Exclude<ERC20BridgeSource, ERC20BridgeSource.Native>;
     makerAmount: BigNumber;
     takerAmount: BigNumber;
+    fillData?: FillData;
 }
 
 export interface MultiHopReportSource {
     liquiditySource: ERC20BridgeSource.MultiHop;
     makerAmount: BigNumber;
     takerAmount: BigNumber;
     hopSources: ERC20BridgeSource[];
+    fillData?: FillData;
```

Comment:
in what cases is `fillData` undefined for `MultiHopReportSource `?

Reply:
This one is a copy-pasta error, it should always be there because it's used in the report code 🤦  Will fix.